engine/api/metrics/metrics.go

```diff
@@ -16,7 +20,9 @@ var (
 )
 
 // Initialize initializes metrics
-func Initialize(c context.Context, DBFunc func() *gorp.DbMap, instance string) {
+func Initialize(c context.Context, DBFunc func() *gorp.DbMap, instance string, instances MinInstances) {
```

Comment:
i would prefer using opencensus/stats lib instead of the having a direct coupling with prometheus

Reply:
done